src/rust/ide/src/model/suggestion_database.rs

```diff
@@ -11,6 +11,7 @@ use language_server::types::SuggestionDatabaseUpdateEvent;
 pub use language_server::types::SuggestionEntryArgument as Argument;
 pub use language_server::types::SuggestionEntryId as EntryId;
 pub use language_server::types::SuggestionsDatabaseUpdate as Update;
+pub use parser::DocParser as DocParser;
```

Comment:
```suggestion
use parser::DocParser;
```
And it should be put into right place according to our Rust Style Guide